docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.5'
+
+services:
+  puppet:
+    hostname: puppet.test
+    image: ${PUPPET_TEST_DOCKER_IMAGE:-puppet/puppetserver-standalone}
+    environment:
+      - PUPPETSERVER_HOSTNAME=puppet.test
+      - PUPPERWARE_ANALYTICS_ENABLED=${PUPPERWARE_ANALYTICS_ENABLED:-false}
+      - DNS_ALT_NAMES=puppet,puppet.test,${DNS_ALT_NAMES:-}
+    dns_search: '.test'
```

Comment:
It's interesting that this works, because I thought what I discovered in my experiments was that this is actually supposed to be:

```yaml
dns_search: test
```